.github/workflows/send_release_notification.yml

```diff
@@ -0,0 +1,16 @@
+name: Send notification on release created
```

Comment:
hmm maybe we should avoid running full PR for just changes in the `.github/` directory..

Reply:
yes, let me add this check also